PocketKit/Sources/PocketKit/Navigation/NavigationSidebarViewController.swift

```diff
@@ -0,0 +1,98 @@
+import UIKit
+import Combine
+import Textile
+
+
+private extension Style {
+    static let title: Style = .header.sansSerif.h7
+}
+
+class NavigationSidebarViewController: UIViewController {
+    private static let layout: UICollectionViewLayout = {
+        var configuration = UICollectionLayoutListConfiguration(appearance: .insetGrouped)
+        configuration.headerMode = .none
+        configuration.backgroundColor = UIColor(.ui.white1)
+        return UICollectionViewCompositionalLayout.list(using: configuration)
+    }()
+    
+    private static let snapshot: NSDiffableDataSourceSnapshot<MainViewModel.AppSection, String> = {
+        var snapshot = NSDiffableDataSourceSnapshot<MainViewModel.AppSection, String>()
+        snapshot.appendSections(MainViewModel.AppSection.allCases)
+        MainViewModel.AppSection.allCases.forEach {
+            snapshot.appendItems([$0.navigationTitle], toSection: $0)
+        }
+        return snapshot
+    }()
+    
+    private lazy var dataSource: UICollectionViewDiffableDataSource<MainViewModel.AppSection, String> = {
+        let registration = UICollectionView.CellRegistration<UICollectionViewListCell, String> { (cell, indexPath, string) in
+            var content = cell.defaultContentConfiguration()
+            content.attributedText = NSAttributedString(string, style: .title)
+            cell.contentConfiguration = content
+        }
+        
+        return UICollectionViewDiffableDataSource<MainViewModel.AppSection, String>(
+            collectionView: collectionView
+        ) { (collectionView, indexPath, string) -> UICollectionViewCell? in
+            let cell = collectionView.dequeueConfiguredReusableCell(
+                using: registration,
+                for: indexPath,
+                item: string
+            )
+            
+            let primaryColor = UIColor(.ui.grey6)
+            
+            let backgroundView = UIView()
+            backgroundView.backgroundColor = primaryColor.withAlphaComponent(0.2)
+            cell.backgroundView = backgroundView
+            
+            let selectedBackgroundView = UIView()
+            selectedBackgroundView.backgroundColor = primaryColor
+            cell.selectedBackgroundView = selectedBackgroundView
+            
+            return cell
+        }
+    }()
+    
+    private lazy var collectionView: UICollectionView = UICollectionView(
+        frame: .zero,
+        collectionViewLayout: Self.layout
+    )
+    private let model: MainViewModel
+    private var subscriptions: Set<AnyCancellable> = []
+
+    init(model: MainViewModel) {
+        self.model = model
+        super.init(nibName: nil, bundle: nil)
+        
+        self.title = "Pocket"
+        
+        collectionView.alwaysBounceVertical = true
```

Comment:
is this not the default?

Reply:
Oddly enough, nope - docs state that the default value is `false` (I did have to double check, as I thought the same thing).